index.js

```diff
@@ -31,12 +31,6 @@ function HtmlWebpackPlugin(options) {
   if(this.options.template.indexOf('!') === -1) {
     this.options.template = require.resolve('./loader.js') + '!' + path.resolve(this.options.template);
   }
-  // Resolve template path
```

Comment:
After I added tests, I ran the code through Istanbul. It showed that this regex never matches anything to replace.
After removing it, all tests still work. Could you give an example when it would replace anything?